index.js

```diff
@@ -8,6 +8,8 @@ if (typeof window !== 'undefined' && window.PouchDB) {
   PouchPromise = typeof global.Promise === 'function' ? global.Promise : require('lie');
 }
 
+var clone = require('lodash.clone');
```

Comment:
We actually have a special module called `pouchdb-exend`. We use it because other ones gave us errors related to using Ember with IndexedDB.